src/App.js

```diff
@@ -1,23 +1,17 @@
-import logo from './logo.svg';
-import './App.css';
+import { Route, Routes } from 'react-router-dom';
+import Layout from './components/layout/Layout';
+import SignIn from './components/signin/SignIn';
+import Order from './components/order/Order';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className='App'>
+      <Routes>
+        <Route path='/' element={<Layout />}>
+          <Route path='' element={<Order />}/>
```

Comment:
why there's no route given in the path?

Reply:
I've not specified any routes because I've to render Order component when a user access application or simply he/she accesses `/` path. This way react will render Order component whenever user accesses `/` path

Reply:
I've found another solution of not specifying any path at all (I still haven't tried this solution).

Reply:
Okay, then what's the difference between `/` and single qoutes`''` in path element above?

Reply:
Basically `/` is used to specify root path, the element which will be rendered an application is accessed. `''` is a default route and in my case it is child of `/` route so it will be rendered along with root element.